source/v4/teams.yaml

```diff
@@ -96,6 +125,45 @@
         '403':
           $ref: '#/responses/Forbidden'
 
+  '/teams/{team_id}/members':
+    get:
+      tags:
+        - teams
+      summary: Get a page of a list of team members based on query string parameters - team id, page and per page.
```

Comment:
Same here, can we make this `Get team members` and then move the extra explanation into the description below?